pysisyphus/calculators/Dimer.py

```diff
@@ -352,13 +353,31 @@ def set_N_raw(self, coords):
             bond_modes = [self.get_bond_mode(bond, coords)
                           for bond in self.bonds]
             N_raw = np.sum(bond_modes, axis=0)
-        # Normalize N_raw
+        # Make N_raw translationally invariant and normalize
         self.N = N_raw
         # Now we keep the normalized dimer orientation
         self.N_raw = self.N
 
         self.log(f"Initial orientation:\n\t{self.N}")
 
+    def make_translationally_invariant(self, coords):
+        # Sum vector components over cartesian directions (x,y,z)
+        # Sum over each direction should equal zero if translationally invariant
+        sum = np.zeros(3)
+        for direction, component in enumerate(coords):
+            sum[direction % 3] += component
+        average = sum / (coords.size / 3)
```

Comment:
The first loop can be avoided by using the built-in method `mean()` on a reshaped `coords` array along
the first axis (0).
```suggestion
        average = coords.reshape(-1, 3).mean(axis=0)
```